packages/aggregator/src/__tests__/analytics/aggregateAnalytics/aggregations/sumPreviousPerPeriod.test.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { PERIOD_TYPES } from '@tupaia/utils';
+import { sumPreviousPerPeriod } from '../../../../analytics/aggregateAnalytics/aggregations';
+
+const { DAY } = PERIOD_TYPES;
+const { YEAR } = PERIOD_TYPES;
+
+describe('sumPreviousPerPeriod()', () => {
+  const testData = [
+    [
+      'should sum across periods',
+      [
+        [
+          [
```

Comment:
Unfortunately I believe that the deeply nested data here will make the test cases harder to read in a parameterized structure... I suggest using the simple `it` blocks (no parameterization).

As in the previous comment, `.toIncludeSameMembers` can be used for the match